brambling/templates/brambling/event/organizer/_refund_form_inner.html

```diff
@@ -0,0 +1,71 @@
+{% load floppyforms zenaida brambling_forms %}
+
+{% with field=form.items bought_items=form.fields.items.queryset.all name=form.items.html_name %}
+
+	{% include "brambling/forms/_top_error_list.html" %}
+
+	{% if bought_items.count > 0 %}
+		<div class="tooltip-wrapper">
+			<div class="list-group">
+				{% if field.errors %}
+					<div class="list-group-item list-group-item-danger">
+						{% for error in field.errors %}
+							{{ error }}
+							{% if not forloop.last %}<br />{% endif %}
+						{% endfor %}
+					</div>
+				{% endif %}
+				{% for item in bought_items %}
+					<div class="list-group-item {% if item.status = 'refunded' %}disabled tipped{% endif %}" {% if item.status = 'refunded' %}title="Refunded" data-placement="left" data-container="body"{% endif %}>
+						<label for="id_refund_{{ item.pk }}" class="row label-block">
+							<div class="col-xs-1 text-center">
+								<input
+									type="checkbox"
+									{% if item.pk|stringformat:"s" in form.items.value or item.status = 'refunded' %}
```

Comment:
So. Fun fact. If you've submitted a value, form.items.value might be a list of strings (which I believe is why this was added) but on initial page load form.items.value is a list of integers. :-D